modules/ingester/ingester.go

```diff
@@ -342,7 +342,7 @@ func (i *Ingester) replayWal() error {
 			return 0, 0, err
 		}
 		return start, end, nil
-	}, log.Logger)
+	}, i.cfg.MaxBlockDuration, log.Logger)
```

Comment:
Yeah agree this is a hard spot but understand the reasons, and this is reasonable.  Maybe a comment ?